subprojects/test-kit/src/integTest/groovy/org/gradle/testkit/runner/GradleRunnerCaptureOutputIntegrationTest.groovy

```diff
@@ -57,8 +59,8 @@ class GradleRunnerCaptureOutputIntegrationTest extends BaseGradleRunnerIntegrati
 
         // isn't empty if version < 2.8 or potentially contains Gradle dist download progress output
         if (isCompatibleVersion('2.8') && !crossVersion) {
-            assert stdStreams.stdOut.empty
-            assert stdStreams.stdErr.empty
+            assert normalize(stdStreams.stdOut).empty
+            assert normalize(stdStreams.stdErr).empty
```

Comment:
:o: Why is this change required?

Reply:
This test will fail on Java 7 because stdout contains a deprecation warning. 

For the tests which are `out.contains('something')`, everything gonna be okay, but for the tests which are `out=='something'` or `out=~/some exp/`, they would fail because the extra deprecation warning.